pkg/kubelet/kubelet.go

```diff
@@ -1853,6 +1860,24 @@ func (kl *Kubelet) syncLoopIteration(configCh <-chan kubetypes.PodUpdate, handle
 				kl.cleanUpContainersInPod(e.ID, containerID)
 			}
 		}
+	case eventDispatcherEvent := <-kl.eventDispacherEventChannel:
+		switch eventDispatcherEvent.Type {
+		case cm.UPDATE_ISOLATOR_LIST:
+			node, err := kl.GetNode()
+			if err != nil {
+				glog.Errorf("cannot achieve node: %q", err.Error())
+				break
+			}
+			if len(eventDispatcherEvent.Body) > 0 {
+				node.ObjectMeta.Labels["external-isolators"] = eventDispatcherEvent.Body
+			} else {
+				delete(node.ObjectMeta.Labels, "external-isolators")
+			}
+			_, err = kl.kubeClient.Core().Nodes().Update(node)
+			if err != nil {
+				glog.Errorf("cannot update node information: %q", err.Error())
```

Comment:
Suggestion to add context: "unable to update list of registered isolators in node: %q"